packages/core/src/Account.ts

```diff
@@ -304,9 +304,6 @@ export class Account extends TypedEventEmitter<Events> {
       // initialize schedulers for pending mls proposals once client is initialized
       await this.service.mls.checkExistingPendingProposals();
 
-      // initialize schedulers for renewing key materials
-      this.service.mls.checkForKeyMaterialsUpdate();
```

Comment:
Now consumer will be the one that initiates intervals and provides the list of `groupIds`